### PR TITLE
Add a two-paragraph overview to main page.

### PR DESCRIPTION
**Pull request by Peter-Easton**
I borrowed from the sponsorship page and cross-referenced from some of the subproject pages where I could. Some of it I had to condense down a little bit. I thought that it might help.

Thought I'd try to address this issue in paragraph form. https://github.com/GrapheneOS/grapheneos.org/issues/27

### CODE REVIEW

**Review by thestinger — CHANGES_REQUESTED** [review]
Can you rebase this and incorporate it into the new main page content?

**Comment by Peter-Easton** [comment]
There we go, I rebased it and changed all the `<i>` tags to `<em>` tags!

File: static/index.html [hunk 1]
```diff
@@ -84,6 +84,56 @@ <h1 id="grapheneos">
             href="https://attestation.app/about">about page</a> and <a
             href="https://attestation.app/tutorial">tutorial</a>. These also support other
             operating systems.</p>
+
+            <h2 id="early-stage-of-development">
```
Comment (thestinger) [inline static/index.html]: This section got removed by the recent changes so you should remove it too:

https://github.com/GrapheneOS/grapheneos.org/commit/516afd143426e27dbc18fb1dd50ba24504f2f73d

I think you mixed up the merge conflict.

File: static/index.html [hunk 2]
```diff
@@ -84,6 +84,56 @@ <h1 id="grapheneos">
...
+            additional documentation and tutorials in the future along with coverage of various
+            software, firmware and hardware privacy/security topics.</p>
+            <p>GrapheneOS is being supported with funding and developers from various companies
+            and other organizations interested in contributing to this shared base for a feature
+            rich private and secure mobile operating system able to run many existing
+            applications. It will take more time to organize and deploy these resources in order
+            for the project to have a strong development team with proper infrastructure behind
+            it.</p>
+
+            <h3 id="overview">
```
Comment (thestinger) [inline static/index.html]: This should probably be integrated above into the existing overview. That's what I meant in terms of integrating it into the current page.

**Review by thestinger — CHANGES_REQUESTED** [review]

File: static/index.html (hunk 2 above)
Comment (Peter-Easton) [reply static/index.html]: Whoops! Sorry, one moment, will get it.

File: static/index.html (hunk 1 above)
Comment (Peter-Easton) [reply static/index.html]: Whoops! I'll try resetting this one and trying it again.

Comment (Peter-Easton) [reply static/index.html]: I think I got it!

**Comment by thestinger** [comment]
I added my own take on this to the main page. If there's anything I left out, feel free to make another pull request. I'll try to work in a mention of Vanadium.

**Comment by Peter-Easton** [comment]
Okay, got it!

**Pull request closed without merging**